src/NationalInstruments.Analyzers/Properties/Resources.resx

```diff
@@ -382,20 +382,20 @@
     <value>Error reading file</value>
     <comment>The title of the diagnostic.</comment>
   </data>
-  <data name="NI1800_Message" xml:space="preserve">
-    <value>{0} is not an approved namespace. If this namespace is valid, please add it to the approved namespaces list - BuildTools\CodeAnalysisRuleSet\NI1800_ApprovedNamespaces.txt.</value>
+  <data name="NI1800_MessageFormat" xml:space="preserve">
```

Comment:
Why did you append 'Format'? That doesn't match the style of the others

Reply:
Reverted the change